src/Sylius/Bundle/ApiBundle/DataProvider/ProductCollectionDataProvider.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\DataProvider;
+
+use ApiPlatform\Core\DataProvider\CollectionDataProviderInterface;
+use ApiPlatform\Core\DataProvider\RestrictedDataProviderInterface;
+use Sylius\Bundle\ApiBundle\Serializer\ContextKeys;
+use Sylius\Component\Core\Model\ProductInterface;
+use Sylius\Component\Core\Repository\ProductRepositoryInterface;
+use Webmozart\Assert\Assert;
+
+final class ProductCollectionDataProvider implements CollectionDataProviderInterface, RestrictedDataProviderInterface
+{
+    /** @var ProductRepositoryInterface */
+    private $productRepository;
+
+    public function supports(string $resourceClass, string $operationName = null, array $context = []): bool
+    {
+        return is_a($resourceClass, ProductInterface::class, true) && $operationName === "";
```

Comment:
```suggestion
        return is_a($resourceClass, ProductInterface::class, true) && $operationName === "product_latest";
```

Reply:
```suggestion
        return is_a($resourceClass, ProductInterface::class, true) && $operationName === "get_latest";
```

Reply:
okey, but for now i prefer name "get_latest"